src/obs_inv_utils/inventory_table_factory.py

```diff
@@ -8,13 +9,27 @@
 from sqlalchemy.orm import relationship, backref
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import sessionmaker
+from dotenv import load_dotenv
+
+load_dotenv()
 
-OBS_SQLITE_DATABASE = 'sqlite:///observations_inventory.db'
 OBS_INVENTORY_TABLE = 'obs_inventory'
 CMD_RESULTS_TABLE = 'cmd_results'
 OBS_META_NCEPLIBS_BUFR_TABLE = 'obs_meta_nceplibs_bufr'
+OBS_DATABASE = ''
+OBS_SQLITE_DATABASE = 'sqlite:///observations_inventory.db'
```

Comment:
what happens if you want to override the default observations_inventory.db file

Reply:
can you elaborate on what you mean here? are you saying you'd want a different file name but still sqlite?

Reply:
Ok, added the functionality but it'll default to observation_inventory.db if not provided.